crates/sui-indexer/src/handlers/pruner.rs

```diff
@@ -71,9 +65,11 @@ impl Pruner {
                         table_name, max_epoch, max_partition
                     );
                 }
-                // drop partitions if pruning is enabled afterwards, where all epochs before min_epoch
-                // would have been pruned already if the pruner was running.
-                for epoch in *min_partition..min_epoch {
+                for epoch in *min_partition..max_epoch.saturating_sub(self.epochs_to_keep - 1) {
```

Comment:
We could use last seen max epoch instead of max epoch from store here to make it clearer that pruner is using its own locally tracked values but tiny suggestion

Reply:
They are always equal though. Or is this more of a which variable to use?

Reply:
Yeah just a matter of which variable to use